.aws/src/main.ts

```diff
@@ -320,18 +329,6 @@ class AnnotationsAPI extends TerraformStack {
             },
           ],
         },
-        {
```

Comment:
Is Xray removed from the code base?

Reply:
@bassrock yep! removed it in the previous PR